File: src/main/java/dev/slne/discord/exception/ticket/member/TicketRemoveMemberException.java
package dev.slne.discord.exception.ticket.member;

import dev.slne.discord.exception.DiscordException;
import java.io.Serial;

public class TicketRemoveMemberException extends TicketMemberException {

  @Serial
  private static final long serialVersionUID = -196685204706749789L;

  public TicketRemoveMemberException(String message) {
    super(message);
  }

  public TicketRemoveMemberException(String message, Throwable cause) {
    super(message, cause);
  }

  public TicketRemoveMemberException(DiscordException exception) {
    super(exception);
  }
}

File: src/main/kotlin/dev/slne/discord/exception/ticket/member/TicketRemoveMemberException.kt
package dev.slne.discord.exception.ticket.member

import dev.slne.discord.exception.DiscordException
import java.io.Serial

class TicketRemoveMemberException : TicketMemberException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(exception: DiscordException) : super(exception)

    companion object {
        @Serial
        private val serialVersionUID = -196685204706749789L
    }
}
